static/describo/zenodo-describo-profile.json

```diff
@@ -0,0 +1,133 @@
+{
+    "metadata": {
+        "name": "Zenodo Describo Profile",
+        "version": 0.1,
+        "description": "A profile for InvenioRDM reposiory",
+        "warnMissingProperty": true
+    },
+    "layouts": {
+        "Dataset": [
+            { "name": "Basic Information", "description": "Basic metadata.", "inputs": ["title", "author", "datePublished", "license"] }
+        ]
+    },
+    "classes": {
+        "Dataset": {
+            "definition": "override",
+            "subClassOf": [],
+            "inputs": [
+                {
+                    "id": "https://schema.org/name",
+                    "definition": "override",
+                    "name": "name",
+                    "label": "Title",
+                    "type": [
+                        "Text"
+                    ],
+                    "required": true,
+                    "multiple": false,
+                    "help": "Title of the deposit."
+                },
+                {
+                    "id": "https://schema.org/author",
+                    "definition": "override",
+                    "name": "author",
+                    "label": "Authors",
+                    "type": [
+                        "Person",
+                        "Organization"
+                    ],
+                    "required": true,
+                    "multiple": true,
+                    "help": "Authors of the deposit."
+                },
+                {
+                    "id": "https://schema.org/Date",
+                    "definition": "inherit",
+                    "name": "datePublished",
+                    "label": "Publication Date",
+                    "type": [
+                        "Date"
+                    ],
+                    "required": false,
```

Comment:
This is required in RDM, just that the deposit form has a "right now" as default value. But there is no default at Marshmallow level, see [here](https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/services/schemas/metadata.py#L343). So it will fail if not sent. Can Describo provide a "default" as we do in the deposit form?

Reply:
Defaults are not available for describo profiles. Perhaps`required` can be used here